### ISSUE TITLE
PENDING ISSUES NOT RESOLVED IN 2.1.1. (some came from the BETA of 2.0)

### ISSUE THREAD

**Issue by jjiglesiasg** [facebook/Rapid#1102]
### Description

a) The UNDO feature during an edition delete ALL THE NODES in a sequence NOT just tha last one.
b) Water features or Waterways of any kind if they are Intermittent are depicted in the SAME form of a NON intermittent waterway or water body.
c) For unknown reason working from MS Edge if you close the app without saving, once opened again the app. The locally saved data do not load, and start in an endless loop. So eventually you loose all the "locally saved data".
d) Way (any kind) over intermitent waterway DO NOT PRODUCE an alert neither there is a solution.
e) Without the Maxar imagery. All AI suggested data seems to be offset several meters, in comparison with Bing, MapBox, Esri and other Imagery providers.
f) The Changesets comments are Not memorized, and you need to type again the whole comment. That's not practical at all.
g) The limit of AI suggested data per Changeset is too low if you are adding buildings.

### Version

2.1.1.

### What browser are you seeing the problem on? What version are you running?

Version 116.0.5845.141 (Official Build) (64-bit)

### Steps to reproduce

Self explained before

### The browser URL at the time you encountered the bug

https://rapideditor.org/rapid#background=Bing&datasets=fbRoads,msBuildings&disable_features=boundaries&map=16.60/-17.97429/-63.27675

**Comment by bhousel** [comment]
Yep, these are all duplicates I think?    It doesn't help us to have them tracked in 2 places.

**Comment by jjiglesiasg** [comment]
These items were supposed to be "Fixed" at Version 2.1.1. several came from Beta 2.0. and they are STILL not fixed
